scio-core/src/main/scala/com/spotify/scio/coders/JPathSerializer.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017 Spotify AB.
```

Comment:
FYI you can use a var in Intellij "Copyright" config: `$today.year`

Reply:
👍 cool I'll update my intellij